internal/checker/printer.go

```diff
@@ -85,6 +85,99 @@ func (c *Checker) SourceFileWithTypes(sourceFile *ast.SourceFile) string {
 	return p.string()
 }
 
+func (c *Checker) GetQuickInfoAtLocation(node *ast.Node) string {
+	symbol := c.GetSymbolAtLocation(node)
+	if symbol != nil && symbol.Flags&ast.SymbolFlagsAlias != 0 {
+		symbol = c.resolveAlias(symbol)
+	}
+	if symbol == nil || symbol == c.unknownSymbol {
+		return ""
+	}
+	flags := symbol.Flags
+	if flags&ast.SymbolFlagsType != 0 && !ast.IsInExpressionContext(node) {
+		// If the symbol has a type meaning and we're not in an expression context, remove any value meanings
+		flags &^= ast.SymbolFlagsValue
+	}
+	p := c.newPrinter(TypeFormatFlagsNone)
```

Comment:
Presumably this is just temporary given I'm sure we'll want to stick this in the LS proper at some point?

Reply:
Yes, pretty much everything in `printer.go` is temporary.